### ISSUE TITLE
Updating Software has encountered a problem

### ISSUE THREAD

**Issue by Klaboe** [ecd-plugin/ecd#78]
When trying to update from 3.2.0 to 3.2.1 i get this error:

```An error occurred while collecting items to be installed
session context was:(profile=<profile_redacted>, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
No repository found containing: osgi.bundle,org.sf.feeling.decompiler,3.2.0.202104251601
No repository found containing: osgi.bundle,org.sf.feeling.decompiler.cfr,3.2.0.202104251601
No repository found containing: osgi.bundle,org.sf.feeling.decompiler.jad,3.2.0.202104251601
No repository found containing: osgi.bundle,org.sf.feeling.decompiler.jd,3.2.0.202104251601
No repository found containing: org.eclipse.update.feature,org.sf.feeling.decompiler.cfr,3.2.1.202104251601
No repository found containing: org.eclipse.update.feature,org.sf.feeling.decompiler,3.2.1.202104251601
No repository found containing: org.eclipse.update.feature,org.sf.feeling.decompiler.jad,3.2.1.202104251601
No repository found containing: org.eclipse.update.feature,org.sf.feeling.decompiler.jd,3.2.1.202104251601
```

I have tried uninstalling and install again from https://ecd-plugin.github.io/update . This only gives 3.2.0 (and also 3.1.1 for some reason)

OS: Win10
Eclipse version: 2021-03 (4.19.0)
Eclipse vm: C:/Program Files/AdoptOpenJDK/jdk-15.0.1.9-hotspot/bin/javaw.exe
PluginInfo: 
  FeatureName: Enhanced Class Decompiler
  Version: 3.2.0.202102151828
  Feature Id: org.sf.feeling.decompiler

**Comment by jpstotz** [comment]
I am sorry for the mess. The Eclipse site builder system really drives me crazy as have to manually update and keep track of version numbers at a dozen places. And there wasn't a full description how to build new versions when I took over the project. 

Therefore those inconsistency are present in the update repository.

**Comment by jpstotz** [comment]
I fixed a lot of version numbers, dependencies and so on and released the changes as v3.2.2. Hopefully now Eclipse should be able to resolve all dependencies and download the appropriate features and plugins.
